scripts/scil_normalize_connectivity.py

```diff
@@ -174,8 +176,11 @@ def main():
                 factor_list.append(np.count_nonzero(
                     atlas_data == label) * voxels_vol)
             else:
-                factor_list.append(approximate_surface_node(
-                    atlas_data, label) * voxels_sur)
+                if np.count_nonzero(atlas_data == label):
+                    factor_list.append(approximate_surface_node(
+                        atlas_data, label) * voxels_sur)
+                else:
+                    factor_list.append(0)
```

Comment:
Is not that going to divide by 0 right after? Which will put an infinite value in the matrix, which will be hard to visualize?

If possible, later I would catch the 0 and put -1 in the matrix. Which will be easy to spot while not screwing up the vizualisation

Reply:
it's divided by >0.001 do you still want to put -1 ?

Reply:
In fact, the behavior should be the same at the volume normalization, no? @frheault you were already doing the np.nonzero_count for the above if. We should have the same behavior for both.

Reply:
Ok when I looked at the raw file I didn't see a few line. Forget about my comments... SOrry